apps/web/src/components/Composer/TextEditor/EmojiPicker.tsx

```diff
@@ -0,0 +1,73 @@
+import type { Emoji } from '@hey/types/misc';
+
+import cn from '@hey/ui/cn';
+import { useEditor } from 'prosekit/react';
+import {
+  AutocompleteItem,
+  AutocompleteList,
+  AutocompletePopover
+} from 'prosekit/react/autocomplete';
+import { useState } from 'react';
+
+import type { TextEditorExtension } from './extension';
+
+import { useEmojiQuery } from './useEmojiQuery';
+
+const EmojiItem = ({
+  emoji,
+  onSelect
+}: {
+  emoji: Emoji;
+  onSelect: VoidFunction;
+}) => {
+  return (
+    <AutocompleteItem
+      className="focusable-dropdown-item m-2 block cursor-pointer rounded-lg p-2 outline-none"
+      onSelect={onSelect}
+    >
+      <div className="flex items-center space-x-2">
+        <span className="text-base">{emoji.emoji}</span>
+        <span className="text-sm capitalize">{emoji.aliases[0]}</span>
+      </div>
+    </AutocompleteItem>
+  );
+};
+
+const EmojiPicker = () => {
```

Comment:
It's not working for me until i type the third character

Reply:
We wan't something similar like github editor 🙇🏼 has with emoji search

Reply:
Done.

<img width="383" alt="image" src="https://github.com/heyxyz/hey/assets/24715727/7a1b364b-ade3-4083-a3a3-748025ddf31b">

I thought you want to start showing emoji when you have some characters typed based on the existing behavior in another emoji picker ([code link](https://github.com/ocavue/hey/blob/680f93876e2fd56b6e4c8fe1ae89dfd2fa77703e/apps/web/src/components/Shared/EmojiPicker/List.tsx#L36)). I've changed it and now it works exactly like Github.